build.sbt

```diff
@@ -56,10 +56,10 @@ lazy val saulExamples = (project in file("saul-examples")).
   settings(
     name := "saul-examples",
     libraryDependencies ++= Seq(
-      "edu.illinois.cs.cogcomp" % "illinois-nlp-pipeline" % cogcompPipelineVersion withSources,
-      "edu.illinois.cs.cogcomp" % "illinois-curator" % cogcompNLPVersion,
-      "edu.illinois.cs.cogcomp" % "illinois-edison" % cogcompNLPVersion,
-      "edu.illinois.cs.cogcomp" % "illinois-corpusreaders" % cogcompNLPVersion,
+      "edu.illinois.cs.cogcomp" % "illinois-nlp-pipeline" % cogcompPipelineVersion exclude("org.slf4j", "slf4j-log4j12") withSourcesxc,
+      "edu.illinois.cs.cogcomp" % "illinois-curator" % cogcompNLPVersion exclude("org.slf4j", "slf4j-log4j12"),
+      "edu.illinois.cs.cogcomp" % "illinois-edison" % cogcompNLPVersion exclude("org.slf4j", "slf4j-log4j12"),
+      "edu.illinois.cs.cogcomp" % "illinois-corpusreaders" % cogcompNLPVersion exclude("org.slf4j", "slf4j-log4j12"),
```

Comment:
hmm not sure. I need to have a closer look.... 
I will do it on Friday probably.